src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx

```diff
@@ -1,26 +1,46 @@
-import { CHANGED_ANNOTATION_COLUMNS } from 'app/pages/newsPage/NewsPageContent';
+import {
+  CHANGED_ANNOTATION_LEVEL_COLUMNS,
+  CHANGED_ANNOTATION_DRUG_COLUMNS,
+} from 'app/pages/newsPage/NewsPageContent';
 import { SimpleTable, SimpleTableRow } from 'app/components/SimpleTable';
 import { Row } from 'react-bootstrap';
 import React from 'react';
 import _ from 'lodash';
 
 import mainstyle from './main.module.scss';
 
+export enum AnnotationColumnHeaderType {
+  LEVEL,
+  DRUG,
+}
+
 export const ChangedAnnotationListItem = (props: {
   title?: string;
   data: SimpleTableRow[];
+  columnHeaderType?: AnnotationColumnHeaderType;
 }) => {
   let longestRow = 0;
   if (props.data.length > 0) {
     longestRow = _.sortBy(props.data, row => -row.content.length)[0].content
       .length;
   }
+
+  let annotationColumnHeader = undefined;
+  switch (props.columnHeaderType) {
+    case AnnotationColumnHeaderType.DRUG:
+      annotationColumnHeader = CHANGED_ANNOTATION_DRUG_COLUMNS;
```

Comment:
You need a break statement here. I'm surprised it still works in the drug table?

Reply:
Good catch, I missed this when setting the default to the level headers